src/app/components/trips/trip-detail/trip-detail.component.html

```diff
@@ -1,3 +1,7 @@
+<div [routerLink]="['/user',(userId$ | async)]">
+  Go To User Profile
+</div>
```

Comment:
@zeus999  make it a button or link or something with the name of the traveller, text inside div is very experimental, please don't send a PR with such code.